packages/utilities/psammead-storybook-helpers/README.md

```diff
@@ -0,0 +1,36 @@
+# psammead-storybook-helpers - [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-assets.svg)](https://www.npmjs.com/package/@bbc/psammead-assets) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
+
+This package provides a collection of common values that are used in storybook by the Psammead components.
+
+## Exports
+
+`text-variants` - A list of text samples in different languages.
+`input-provider` - A function that provides support for selecting between the `text` and `select` storybook knobs.
+
+## Installation
+
+```jsx
+npm install @bbc/psammead-storybook-helpers --save-dev
+```
+
+## Usage
+
+```jsx
+import { LANGUAGE_VARIANTS } from '@bbc/psammead-storybook-helpers/text-variants';
```

Comment:
These should either be defaults, or exported from the base level (directly out the dist folder like our components)